github_helper.py

```diff
@@ -0,0 +1,203 @@
+"""github_helper provides several methods to help interface with Github.
+"""
+
+import sys
+import urlparse
+import dateutil.parser
+import requests
+from requests.exceptions import HTTPError, Timeout, RequestException
+
+GITHUB_API_ROOT = 'https://api.github.com'
+GITHUB_REPO_FMT_URL = GITHUB_API_ROOT + '/repos/{0}/{1}/'
+GITHUB_PULLS_ENDPOINT = 'pulls'
+
+BOT_NAME = 'merge-bot'
+GITHUB_SECRET = '../../github_auth/mergebot.secret'
+
+
+class GithubHelper(object):
+    """GithubHelper is a lightweight authenticated wrapper for the Github API.
+    """
+
+    def __init__(self, org, repo):
+        self.repo_url = GITHUB_REPO_FMT_URL.format(org, repo)
+        with open(GITHUB_SECRET, 'r') as key_file:
+            self.bot_key = key_file.read().strip()
+        _print_flush('Loaded key file.')
```

Comment:
That's only for class-private though - _print_flush is package-private instead is my understanding.